generator/template/vue-sections/plugins/sections.js

```diff
@@ -1,5 +1,5 @@
 import Vue from "vue";
-import Sections from "@geeks-solutions/vue-sections";
+import Sections from "@geeks.solutions/vue-sections";
```

Comment:
if we can only install the nuxt-sections plugin (so the sections work correctly, ie comment on the Wysiwyg component)

Reply:
Updated the README file